language/move-model/src/pragmas.rs

```diff
@@ -287,6 +287,10 @@ pub const CONDITION_CHECK_ABORT_CODES_PROP: &str = "check";
 /// enabled disabled by the disable_invariant_in_body pragma
 pub const CONDITION_SUSPENDABLE_PROP: &str = "suspendable";
 
+/// A property that can be attached to a loop invariant to indicate that the loop needs to
+/// be unrolled to a certain depth, a typical relaxation in bounded model checking.
```

Comment:
This allows controlling the unrolling on the finest level of granularity, by loop. But can we also have a pragma (perhaps in a 2nd PR) which can be attached to function or module level, and applies to all loops? This avoids having to look at every single loop.